### ISSUE TITLE
Radarr won't start again on Synology after reboot

### ISSUE THREAD

**Issue by christiga** [Radarr/Radarr#1728]
**Description:**
I installed Radarr this week on my DS216play. This morning I had to reboot the NAS. Radarr wouln't start anymore.

Tried : 
- start from package center. Still 'stopped'
- run from command : sudo /var/packages/radarr/scripts/start-stop-status start
"Starting Radarr ..." is appearing but Radarr still not started. No error.
- changed user from the start-stop-status script to no avail

**Radarr Version:**
20170226-2 (installed from package center)

**Logs:**
No logs have been created since reboot

Is there a command to run radarr directly ?
Is there a way to have some information on errors when running the start script?

Thank you !

**Comment by galli-leo** [comment]
Are you sure no logs are created in the AppData dir? Because if that's the case, I think the startup script has a problem.

**Comment by myellen** [comment]
The way I fixed this was to 

1. Uninstall the radarr package through package center.
2. In DSM, Control Panel > Users and manually create a new user named "radarr" with read/write permissions to the download and media folders.
3. Reinstall radarr from package center.

I've used the same solution for other packages like sonarr and sabnzbd too.

**Comment by christiga** [comment]
galli-leo : yes I'm sure, the last logs are dated before the reboot, I'll check them again.

myellen : yes, that's what I do since DSM 6 for every package from synocommunity... I'll try to uninstall and reinstall, I think it will erase the Radarr db, right?

**Comment by christiga** [comment]
Ok I uninstalled and reinstalled Radarr, it works now.
I also uninstalled mono (the package from syno) and kept the synocommunity one, I don't know why I had both.
Weird thing though : on the UPNP router configuration ("external access" on the syno configuration or whatever name it is in English, mine is in French), "Sonarr" line is renamed to "Radarr", maybe there is still something that goes wrong when the 2 of them are installed.
How can I restore my data (I have a backup) ? I will search the wiki anyway.
